src/test/java/klerer/gameoflife/GameOfLifeTest.java

```diff
@@ -15,7 +15,7 @@ public void testNextGeneration() {
                 {0, 1, 0}
         };
 
-        GameOfLife gameOfLife = new GameOfLife(initialGrid);
+        GameOfLife gameOfLife = new GameOfLife(50, 50);
```

Comment:
You changed this to be 50x50 in your test but the initialGrid is 3x3

Reply:
I fixxed that but forgot to push it. its still failing. i will push it now